### DISCUSSION TITLE
Lock rotation axis/restrict axis to certain degrees & "fold-out" annotations

### DISCUSSION THREAD

**Discussion by warmadsen** [google/model-viewer#1988] [Q&A]
Hello!  With model-viewer, how exactly do i lock the camera-orbit to for example horizontal orbit. For example I don't want the camera to be able to orbit vertically. I am not very good in coding, so if it is possible, i need an exact guide xD

Another question: is it possible to make the annotations "fold out" when you press the button? Right now they are just floating around and if the model is small, but has a lot of annotations, it becomes quite clustered :-)

Thanks, and sorry for disturbing! I love the work you guys are doing!

**Comment by elalish** [comment]
For orbit, you'll want [`min/max-camera-orbit`](https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-maxCameraOrbit), so something like `min-camera-orbit="auto 75deg auto"` and `max-camera-orbit="auto 75deg auto"`.

For folding out annotations, it'll just be standard HTML techniques. Like put a click listener on your hotspot that toggles the `hidden` or `display:none` property of its child element. Consider how the Show Dimensions button works in this example: https://modelviewer.dev/examples/annotations/#dimensions

**Reply by warmadsen** [reply]
Thank you so much for the quick response and the super help! I know what to do now :-)
